tests/AppBundle/EventListener/CommentNotificationListenerTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Tests\AppBundle\EventListener;
+
+use AppBundle\Entity\Comment;
+use AppBundle\Entity\Post;
+use AppBundle\Entity\User;
+use AppBundle\EventListener\CommentNotificationListener;
+use Symfony\Component\EventDispatcher\GenericEvent;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * Unit test for the event listener
+ *
+ * See http://symfony.com/doc/current/book/testing.html#unit-tests
+ *
+ * Execute the application tests using this command (requires PHPUnit to be installed):
+ *
+ *     $ cd your-symfony-project/
+ *     $ ./vendor/bin/phpunit
+ */
+class CommentNotificationListenerTest extends \PHPUnit_Framework_TestCase
+{
+    public function testOnCommentCreated()
+    {
+        $mailerProphecy = $this->prophesize(\Swift_Mailer::class);
+        $urlGeneratorProphecy = $this->prophesize(UrlGeneratorInterface::class);
+        $translatorProphecy = $this->prophesize(TranslatorInterface::class);
+
+        $user = new User();
+        $user->setEmail('bar@foo.fr');
+
+        $post = new Post();
+        $post->setTitle('bar foo');
+        $post->setAuthor($user);
+        $post->setSlug('bar-foo');
+
+        $comment = new Comment();
+        $comment->setPost($post);
```

Comment:
@voronkovich Yes but there is no setter for id. I do not think this is a good way to add it.